### PR TITLE
Introduce Setting to allow/decline telemetry

### PR DESCRIPTION
**Pull request by Tsumiki-Chan**
Introduces a setting which controls whether LSS-Manager is allowed to send Telemetry Data.

Don't stone me if this is completely wrong. This is my first time 😏

### CODE REVIEW

File: src/modules/telemetry/i18n/nl_NL.root.json [hunk 1]
```diff
@@ -14,8 +14,14 @@
 	"info": {
 		"close": "Toon deze melding niet overnieuw",
 		"decline": "Sta het verzamelen van metadata niet toe",
-		"text": "De LSS Manager verzamelt metadata zoals beschreven in de <a href='{wiki}metadata.html' class='lightbox-open'>Wiki</a>.",
+		"text": "De LSS Manager verzamelt metadata zoals beschreven in de <a href='{wiki}metadata.html' class='lightbox-open'>Wiki</a>.<br>U kunt uw akkoord op een later moment altijd herroepen.",
```
Comment (jxn-30) [inline src/modules/telemetry/i18n/nl_NL.root.json:17]: @JRH-1997 @Piet2001 is this good nl_NL translation?

File: src/modules/telemetry/main.ts [hunk 2]
```diff
@@ -117,8 +117,13 @@ export default (LSSM: Vue): void => {
...
                                     .dispatch('storage/set', {
                                         key: NOTE_STORAGE_KEY,
-                                        value: 'declined',
+                                        value: true,
                                     } as StorageSet)
+                                    .dispatch('settings/setSetting', {
+                                        moduleId: MODULE_ID,
+                                        settingId: 'allowTelemetry',
+                                        value: false,
+                                    } as SettingSet)
```
Comment (jxn-30) [inline src/modules/telemetry/main.ts:126]: that will not work ^^

**Review by jxn-30 — CHANGES_REQUESTED** [review]

**Comment by Tsumiki-Chan** [comment]
Better now? 😳

File: src/modules/telemetry/i18n/nl_NL.root.json (hunk 1 above)
Comment (JRH-1997) [reply src/modules/telemetry/i18n/nl_NL.root.json:17]: U kunt uw akkoord op een later moment altijd herroepen.
to:
U kunt uw toestemming ten alle tijden intrekken.

Comment (jxn-30) [reply src/modules/telemetry/i18n/nl_NL.root.json:17]: @Tsumiki-Chan ;)

**Review by jxn-30 — APPROVED** [review]

**Review by Piet2001 — APPROVED** [review]

**Pull request merged**